redi-connect-front/src/pages/app/profile/mentor/ProfileMentor.tsx

```diff
@@ -1,100 +1,104 @@
-      color: 'white',
-      fontSize: '12px',
-      margin: '3px',
-      height: '20px'
-    },
-    personalDescription: {
-      marginTop: theme.spacing(6),
-      marginBottom: theme.spacing(6),
-      overflowWrap: 'break-word'
-    }
-  })
-
-export const ProfileMentor = withStyles(styles)(
-  ({ mentor, classes }: Props) => {
+export const ProfileMentor = (
+  ({ mentor }: Props) => {
     const profile = getRedProfile()
+    {/* {(mentor.numberOfPendingApplicationWithCurrentUser === 0 && profile.userType === 'mentee') && (
+      <ConnectionRequestForm mentorId={mentor.id} />
+    )}
+    {mentor.numberOfPendingApplicationWithCurrentUser > 0 && (
+      <p>You have already applied to this mentor.</p>
+    )} */}
 
     return <>
-      <Grid container spacing={2}>
-        <Grid item xs={12} sm={5}>
-          {/* <Avatar
-            className={classes.avatar}
-            s3Key={mentor.profileAvatarImageS3Key}
-            style={{ width: '100%', height: '20vh' }}
-          /> */}
-        </Grid>
-        <Grid item xs={12} sm={7}>
-          <h3 style={{ fontWeight: 700, fontFamily: 'Roboto' }}>
-            <ProfileName name={`${mentor.firstName} ${mentor.lastName}`} />
-            <ProfileOccupation occupation={mentor.mentor_occupation} />
-            <ProfileWorkPlace workPlace={mentor.mentor_workPlace} />
-            <ProfileLanguages languages={mentor.languages} />
-          </h3>
-        </Grid>
-      </Grid>
-
-      <ProfileAvailableMenteeSlots
-        totalCapacity={mentor.menteeCountCapacity}
-        currentFreeCapacity={mentor.currentFreeMenteeSpots}
-      />
-      <p>I can help you with:</p>
-      {mentor.categories.map(catId => (
-        <CategoryChip
-          key={catId}
-          className={classes.category}
-          categoryId={catId}
-        />
-      ))}
-      <p className={classes.personalDescription}>
-        {mentor.personalDescription}
-      </p>
-      {mentor.expectations && (
-        <>
-          <h4 style={{ marginBottom: 0 }}>Expectations to my mentee:</h4>
-          <p
-            className={classes.personalDescription}
-            style={{ marginTop: '0.3em' }}
-          >
-            {mentor.expectations}
-          </p>
-        </>
-      )}
-
-      {(mentor.numberOfPendingApplicationWithCurrentUser === 0 && profile.userType === 'mentee') && (
-        <ConnectionRequestForm mentorId={mentor.id} />
-      )}
-      {mentor.numberOfPendingApplicationWithCurrentUser > 0 && (
-        <p>You have already applied to this mentor.</p>
-      )}
+      <Columns vCentered breakpoint="mobile">
+        <Columns.Column size={3}>
+          <Avatar />
+        </Columns.Column>
+        <Columns.Column size={9}>
+          <Heading>{`${mentor.firstName} ${mentor.lastName}`}</Heading>
+        </Columns.Column>
+      </Columns>
+      {mentor.personalDescription && <Element className="me__block">
+        <Read title={`about ${mentor.firstName} ${mentor.lastName}`}>
```

Comment:
The read is a good idea however it also may be a good idea to make an about component as we discussed. 
Check ProfileCourse, ProfileName, etc...

Reply:
new component `InfoText` added